generation/3d_ldm/3d_ldm_tutorial.ipynb

```diff
@@ -0,0 +1,1206 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plt.plot(epoch_loss_list)
</pre><p>Let‘s add a markdown for this cell as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
What would we state here?

Reply:
“## Plotting Learning Curves”？